### PR TITLE
Support for Laravel 8

### PR DESCRIPTION
**Pull request by mr4rtur**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,11 +20,11 @@
...
-        "php": "^7.2",
-        "illuminate/database": "^6.0|^7.0"
+        "php": "^7.3",
+        "illuminate/database": "^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0",
-        "orchestra/testbench": "^5.1"
+        "phpunit/phpunit": "^8.4|^9.0",
+        "orchestra/testbench": "^6.0"
```
Comment (stancl) [inline composer.json:28]: Please add a `|` here rather than changing the major version

File: composer.json [hunk 2]
```diff
@@ -20,11 +20,11 @@
         }
     ],
     "require": {
-        "php": "^7.2",
-        "illuminate/database": "^6.0|^7.0"
+        "php": "^7.3",
```
Comment (stancl) [inline composer.json:23]: Laravel 7.x still supports PHP 7.2

File: composer.json [hunk 3]
```diff
@@ -20,11 +20,11 @@
...
+        "illuminate/database": "^8.0"
```
Comment (stancl) [inline composer.json:24]: Again, this should use `|`

**Review by stancl — CHANGES_REQUESTED** [review]

**Comment by stancl** [comment]
Also tests should now run in a matrix for both L7 and L8

File: composer.json [hunk 4]
```diff
@@ -20,11 +20,11 @@
...
     "require": {
-        "php": "^7.2",
-        "illuminate/database": "^6.0|^7.0"
+        "php": "^7.2|^7.3",
+        "illuminate/database": "^6.0|^7.0|^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0",
-        "orchestra/testbench": "^5.1"
+        "phpunit/phpunit": "^8.4|^9.0",
```
Comment (stancl) [inline composer.json:27]: What's the reasoning behind the `^8.4` change?

**Review by stancl — APPROVED** [review]
Looks great!

File: composer.json (hunk 4 above)
Comment (mr4rtur) [reply composer.json:27]: Just wanted to be consistent with Laravel framework (versions 7.x and 8.x).

Comment (stancl) [reply composer.json:27]: I see, makes sense!

**Comment by stancl** [comment]
Thanks a lot!

**Pull request merged**